dev-java/protobuf-java/protobuf-java-3.21.5.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 2008-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Tests not enabled, depend on com.google.truth which is not packaged
+# https://github.com/protocolbuffers/protobuf/blob/v21.5/java/core/pom.xml#L35-L40
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.google.protobuf:${PN}:${PV}"
+
+inherit java-pkg-2 java-pkg-simple
+
+PARENT_PN="${PN/-java/}"
+PARENT_PV="$(ver_cut 2-)"
+PARENT_P="${PARENT_PN}-${PARENT_PV}"
+
+if [[ "${PV}" == *9999 ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/protocolbuffers/protobuf.git"
+	EGIT_SUBMODULES=()
+	EGIT_CHECKOUT_DIR="${WORKDIR}/${PARENT_P}"
+else
+	SRC_URI="
+		https://github.com/protocolbuffers/protobuf/archive/v${PARENT_PV}.tar.gz
+			-> ${PARENT_P}.tar.gz
+	"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+fi
+
+DESCRIPTION="Google's Protocol Buffers - Java bindings"
+HOMEPAGE="
+	https://developers.google.com/protocol-buffers/
+"
+
+LICENSE="BSD"
+SLOT="0/32"
```

Comment:
Subslot for a Java package? Why not simply SLOT="0" ?

Reply:
This "0/X" format is used in BDEPEND for parent package, and it has no impact on consumers side as far as I know.

Reply:
> This "0/X" format is used in BDEPEND for parent package, and it has no impact on consumers side as far as I know.

Checked about what other Java packages might use subslots and it looks like being unique to just only this certain package.
Putting/keeping it here would only confuse other packagers working with [Gentoo Java Packing Policy](https://wiki.gentoo.org/wiki/Gentoo_Java_Packing_Policy#Slotting).
```
$ qgrep -e 'SLOT=.*\/.*' | grep dev-java
dev-java/protobuf-java/protobuf-java-3.19.3.ebuild:SLOT="0/30"
dev-java/protobuf-java/protobuf-java-3.20.1.ebuild:SLOT="0/31"
dev-java/protobuf-java/protobuf-java-9999.ebuild:SLOT="0/31
```